init kicad-exports to auto generate ibom and gerbers

nerdyscout opened the pull request:
using my CI action [kicad-export](https://github.com/nerdyscout/kicad-exports) to sequentially auto update  
```
  pnp/pcb/mobo/bom/ibom.html
  pnp/pcb/piggyback/bom/ibom.html
  feeder/pcb/mobo/bom/ibom.html
```
and generate  zip archive with `gerbers`, `position.csv` and `bom.csv` for each individual PCB on every push/pull_request on master if *.sch or *.kicad has been changed

Generating additional files would be possible (*.step, schematic.pdf/.svg, board_top/_bottom.svg) if needed.

sphawes commented:
hey @nerdyscout sorry for the delay on this! this is insanely cool! thanks for the PR. 

two questions: 
1) once there's a merge into master, this script would immediately add an extra commit directly to master with these changes?
2) getting a schematic pdf output would be incredible for folks that want to take a peek at the design but not clone the whole repo. how can I go about making that change? seems like you have an example config file for it [here](https://github.com/nerdyscout/kicad-exports/blob/master/config/docs.kibot.yaml). is it just a matter of dropping this file into the pcb directory?

nerdyscout commented:
just yesterday I finalized all features which will be part of kicad-exports@v2.2. so far there is one bug left, I suggest you wait merging till I fixed this and updated this pull request.

1) yes. or you could set it up to push to another branch, manually review changes, and then merge to master - whatever you like more.
Actually, with the current pull request, it would add three commits - one for each job/pcb. As I noticed this can fail because rebaseing is strange... I will update the pull request to a single commit. 

2) yes, this is the config to go for schematics. As the pull request includes individual configs for each PCB the schematic output can be merged into the same config.

you could take a look at https://github.com/Open-Smartwatch/kicad-project about how kicad-exports integrates into a existing repository.

nerdyscout commented:
update done :tada: 

this does is how your repository should look like after merging
https://github.com/nerdyscout/index/tree/test

kicad-exports@v2.2 isn't released yet, please be aware this is bleeding edge
but setting this up seemed to work very smooth, I don't expect much changes.
